service/storage_proxy.cc

```diff
@@ -865,8 +865,12 @@ class storage_proxy::remote {
             std::move(pr), oda, rate_limit_info_opt, fence);
     }
 
-    future<> handle_truncate(rpc::opt_time_point timeout, sstring ksname, sstring cfname) {
-        return replica::database::truncate_table_on_all_shards(_sp._db, _sys_ks, ksname, cfname);
+    future<> handle_truncate(rpc::opt_time_point timeout, sstring ksname, sstring cfname, std::optional<frozen_topology_guard> frozen_guard) {
```

Comment:
rpc::optional

Reply:
A new RPC was added in v7. Closing the request because it is obsolete.